webapp/src/Controller/Jury/TeamController.php

```diff
@@ -429,20 +429,8 @@ public function addAction(Request $request): Response
                 // If we do not want to add a user, remove it again
                 $team->removeUser($user);
             } else {
-                // Otherwise, add the team role to it
-                /** @var Role $role */
-                $role = $this->em->createQueryBuilder()
-                    ->from(Role::class, 'r')
-                    ->select('r')
-                    ->andWhere('r.dj_role = :team')
-                    ->setParameter(':team', 'team')
-                    ->getQuery()
-                    ->getOneOrNullResult();
-                $user->addUserRole($role);
-                $user->setTeam($team);
-                // Also set the user's name to the team name
+                // Otherwise, set the user's name to the team name
```

Comment:
Why is the user created at all? I don't think its needed for the normal add page. But thats probably not part of your PR.